server/sonar-process/src/main/java/org/sonar/process/ProcessProperties.java

```diff
@@ -48,15 +49,16 @@
   public static final String PATH_TEMP = "sonar.path.temp";
   public static final String PATH_WEB = "sonar.path.web";
 
+  public static final String SEARCH_CLUSTER_NAME = "sonar.search.clusterName";
```

Comment:
I see below the following comment

> name of ES cluster must always be set, even if clustering of SQ is disabled

I think `EsSettings` class should take property `sonar.cluster.search.clusterName` into account when SQ runs as a cluster, otherwise use a hardcoded value

we would be more consistent in our property naming and would not allow the ES cluster name to be changed in any way when not running as a SQ cluster